test/components/TaskListTest.js

```diff
@@ -163,23 +114,6 @@ describe('TaskList', () => {
       const task = store.getState().tasks[0]
       assert.equal('string', typeof task.archivedAt)
     })
-
-    it('shows tasks again if `updatedAt` is greater than `archivedAt`', () => {
```

Comment:
It should totally be the case. It looks like I tried to move it but never did :sad:. I'll add it back in.